deployments/hardhat/dai/deploy.ts

```diff
@@ -44,9 +45,9 @@ async function makePriceFeed(
 export default async function deploy(deploymentManager: DeploymentManager) {
   const [admin, pauseGuardianSigner] = await deploymentManager.getSigners();
 
-  let dai = await makeToken(deploymentManager, 1000000, 'DAI', 18, 'DAI');
-  let gold = await makeToken(deploymentManager, 2000000, 'GOLD', 8, 'GOLD');
-  let silver = await makeToken(deploymentManager, 3000000, 'SILVER', 10, 'SILVER');
+  let dai = await makeToken(deploymentManager, 10000000, 'DAI', 18, 'DAI');
+  let gold = await makeToken(deploymentManager, 20000000, 'GOLD', 8, 'GOLD');
+  let silver = await makeToken(deploymentManager, 30000000, 'SILVER', 10, 'SILVER');
```

Comment:
Increasing these by a factor of 10 since the `baseMinForRewards` is 1 million, which is already the entire supply of our test DAI token.

Reply:
Although could also lower `baseMinForRewards` for dev configuration...but maybe nicer to keep that one the same as `mainnet`